### PR TITLE
S1 final plan

### PR DESCRIPTION
**Pull request by PapaPwn**

### CODE REVIEW

**Comment by PapaPwn** [comment]
Review to be completed at retro, so that proper priorities and estimates can be set.

File: docs/project-plan.md [hunk 1]
```diff
@@ -3,162 +3,332 @@
...
-4. Releases
-5. Roles
+2. Value Stories
+3. Releases
+4. Roles
+5. Sprints & Meeting Schedule
 
-## 1.0 Product Statement
+## 1.0 - Product Statement
+University is full of hard work and tough challenges. Unfortunately, this experience is made more difficult without a centralised source for course information. 'Smart Course' is our solution to this problem.
```
Comment (DarkPurple141) [inline docs/project-plan.md]: Do we defs not need functional/non functional requirements?

File: docs/project-plan.md [hunk 2]
```diff
@@ -3,162 +3,332 @@
...
-#### Question priority
-As a question answerer I want to quickly see the most relevant or popular questions.
-
-#### Answerer Reputation
-As a reliable contributer I want my answers to be given more weight.
-
-#### Answerer verification
-As someone looking at the answer/s to a question, I want to see which answerers are teachers (lecturers, TAs) for the course and which answerers have done the course (or are currently doing it).
+As a student, I want to be able to ask a question about a course, so that I have access to information I didn't have previously.
+ * Estimate: 6
```
Comment (DarkPurple141) [inline docs/project-plan.md]: Not sure if we can estimate these very well in here, but I spose it's just for show. Agree need to formalise the Stories that make it into the MVP. (There is a release tag).

File: docs/project-plan.md [hunk 3]
```diff
@@ -3,162 +3,332 @@
...
+    * Minimal/incomplete features.
+    * Testing integration of features.
+2. Minimal Viable Product (Week 8/9):
+    * Basic minimal requirements met.
+3. Final Product (Week 12):
+    * Thoroughly tested and polished.
+
+## 4.0 - Roles
+In order to give everyone a taste of the different roles, we will be doing a fortnightly rotation for the 'Scrum Master' and 'Product Owner', whilst everyone is a 'Developer'.
+
```
Comment (DarkPurple141) [inline docs/project-plan.md]: This is good.

File: docs/project-plan.md [hunk 4]
```diff
@@ -3,162 +3,332 @@
...
+    * Scrum Master - Alex
+    * Product Owner - Nuno
+
+## 5.0 - Sprints & Meeting Schedule
+
+### Sprints
+Each sprint is one week long (Thursday to Thursday). The goal for each sprint is to choose a subset of user stories to work on and ideally finish within that sprint. To enable this to occur in an efficient fashion, we have planeed for both standups and sprint reviews.
+
+### Standups
+The goal of the standup is for each team member to answer the follwoing 3 questions:
```
Comment (NunoDasNeves) [inline docs/project-plan.md]: follwoing -> following

**Comment by PapaPwn** [comment]
Still need todo clear definition of MVP and Discuss Technologies used. The rest of the document can be reviewed though.

File: docs/project-plan.md (hunk 4 above)
Comment (PapaPwn) [reply docs/project-plan.md]: will be fixed next push.

**Pull request merged**